### PR TITLE
The memory of indexnode must be 6 times the segment size

### PR DESCRIPTION
**Pull request by xiaocai2333**
Signed-off-by: cai.zhang <cai.zhang@zilliz.com>

issue: #10754

### CODE REVIEW

File: internal/indexcoord/node_manager.go [hunk 1]
```diff
@@ -109,18 +109,18 @@ func (nm *NodeManager) PeekClient(meta Meta) (UniqueID, types.IndexNode) {
...
 		return UniqueID(-1), nil
 	}
-	indexSize, err := estimateIndexSize(dim, meta.indexMeta.Req.NumRows, meta.indexMeta.Req.FieldSchema.DataType)
+	dataSize, err := estimateIndexSize(dim, meta.indexMeta.Req.NumRows, meta.indexMeta.Req.FieldSchema.DataType)
 	if err != nil {
 		log.Warn(err.Error())
 		return UniqueID(-1), nil
 	}
-	nodeID := nm.pq.Peek(indexSize, meta.indexMeta.Req.IndexParams, meta.indexMeta.Req.TypeParams)
+	nodeID := nm.pq.Peek(dataSize*6, meta.indexMeta.Req.IndexParams, meta.indexMeta.Req.TypeParams)
```
Comment (czs007) [inline internal/indexcoord/node_manager.go:117]: define a const value

Comment (xiaocai2333) [reply internal/indexcoord/node_manager.go]: done

**Comment by xiaofan-luan** [comment]
how do we get the number six? is this a magic number?

**Comment by xiaocai2333** [comment]
> how do we get the number six? is this a magic number?

Yes, it is obtained by observing the phenomenon during the test.

**Pull request merged**